### ISSUE TITLE
Change include file order in Cythonizing _cantera.pyx

### ISSUE THREAD

**Issue by thomasfiala** [Cantera/cantera#284]
Currently, the linking file order while Cythonizing _cantera.pyx is something like (output from `scons build`):

```
Cythonizing /Users/tf/Documents/td/cantera/interfaces/cython/cantera/_cantera.pyx
clang++ -o build/temp-py/_cantera2.os -c -fcolor-diagnostics -O3 -g -fPIC -DNDEBUG -I/opt/local/include -I/opt/local/include -I/opt/local/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7 -I/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/numpy/core/include -Iinclude interfaces/cython/cantera/_cantera.cpp
```

This works fine if there is only one instance of Cantera installed. If Cantera headers are installed in an additional location (e.g. by MacPorts in /opt/local/include/cantera/), this may lead to problems: As these files are included before the local `include` directory, the cython interface is built against the wrong headers.

If possible, the cythonizing command should instead look something like:

```
clang++ -o build/temp-py/_cantera2.os -c -fcolor-diagnostics -O3 -g -fPIC -DNDEBUG -Iinclude -I/opt/local/include -I/opt/local/include -I/opt/local/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7 -I/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/numpy/core/include interfaces/cython/cantera/_cantera.cpp
```

**Comment by speth** [comment]
I think this should be fixed in 07f25621f317b47b91aacc27e7d0da84fc19ac44, if you could please confirm.

**Comment by thomasfiala** [comment]
Yes, this appears to have fixed the issue.Thanks!
